aws-frauddetector-eventtype/src/aws_frauddetector_eventtype/helpers/api_helpers.py

```diff
@@ -213,10 +256,19 @@ def call_create_variable(
         "variableType": variable_type,
         "tags": variable_tags,
     }
-    validation_helpers.remove_none_arguments(args)
+    args = validation_helpers.remove_none_arguments(args)
     return frauddetector_client.create_variable(**args)
 
 
+@batch_call_limit(request_attribute_to_split="variable_entries", response_items_to_collect=["errors"], limit=25)
+@api_call_with_debug_logs
+def call_batch_create_variable(frauddetector_client, variable_entries: List[dict], tags: List[dict]):
+    args = {"tags": tags, "variableEntries": variable_entries}
+    args = validation_helpers.remove_none_arguments(args)
+    validation_helpers.check_variable_entries_are_valid(args)
+    return frauddetector_client.batch_create_variable(**args)
```

Comment:
BatchCreateVariable to create 25 variables at once. This enables a drastic reduction in required number of AFD API calls for event types with lots of event variables.